vmdb/app/views/layouts/_edit_to_email.html.haml

```diff
@@ -12,9 +12,11 @@
             = _("None")
           - else
             - if @temp[:email_to].present?
-              - emails = @temp[:email_to].sort { |a, b| a[1] <=> b[1] }.collect { |e| e[0] }
+              - emails = @temp[:email_to]
+                  .sort_by { |_email, display_email| display_name }
+                  .collect { |email, _display_email| email }
             - elsif !@edit[:new][:email][:to].present?
-              - emails = @edit[:new][:email][:to].sort { |a, b| a[1] <=> b[1] }
+              - emails = @edit[:new][:email][:to].sort
```

Comment:
@Fryguy You're right. It's [haml conversion bug](https://github.com/ManageIQ/manageiq/commit/a0bda611cd2f10e805553ea4ae14b3b15b1af730#diff-71212314ed2cc7d063cbb48e2c286506R16). I forgot to remove exclamation mark in that condition